### ISSUE TITLE
Audit Linksharing TLS test suite

### ISSUE THREAD

**Issue by shaupt131** [storj/gateway-mt#242]
### Goal

- We need to make sure the new feature has adequate test coverage in order to guarantee quality for our customers.

### Acceptance Criteria

- All components are end-to-end tested
- The basic flow that QA's test plan tests is what we test automatically
    - https://github.com/storj/storj/issues/5125

****Blocked by https://github.com/storj/gateway-mt/issues/238****

**Comment by shaupt131** [comment]
Test Plan: https://github.com/storj/storj/issues/5125

**Comment by amwolff** [comment]
I think we need to split this issue into two. The changes above I pushed exhaust the first bullet point, and it took some effort to audit testing of all components; I believe the second bullet point is also a significant effort, warranting its issue to track it. What do you think, @ferristocrat?

**Comment by amwolff** [comment]
Superseded by #306.
